src/toast/_libtoast/tod_pointing.cpp

```diff
@@ -131,5 +131,129 @@ void init_tod_pointing(py::module & m) {
 
     )");
 
+    m.def("healpix_pixels",
```

Comment:
It looks like you have duplicated these bindings.  Was that intentional?

Reply:
Nope. Fixed. Thanks!